.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+  - 2.7
+
+services:
+  - docker
+
+before_install:
+  - cd daemon
+
+install:
+  - docker build -t daemon .
+  - docker run -d --name ceph-mon --net=host -v /etc/ceph:/etc/ceph -v /var/lib/ceph:/var/lib/ceph -e MON_IP=127.0.0.1 -e CEPH_PUBLIC_NETWORK=127.0.0.0/8 daemon mon
+  - sudo ../travis-builds/build-osd.sh
+  - docker run -d --name ceph-osd -v /etc/ceph:/etc/ceph -v /var/lib/ceph:/var/lib/ceph daemon directory
```

Comment:
Try to use a OSD_DIRECTORY instead  directory.
`docker run -d --name ceph-osd -v /etc/ceph:/etc/ceph -v /var/lib/ceph:/var/lib/ceph daemon OSD_DIRECTORY `